src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -25,10 +32,25 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
         return opprettFørstegangsbehandling(opprettTilbakekrevingRequest)
     }
 
-    fun opprettBehandlingManuell(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest) : Behandling{
+    fun opprettBehandlingManuell(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest): Behandling {
         TODO("Ikke implementert ennå")
     }
 
+    @Transactional(readOnly = true)
+    fun hentBehandling(eksternBrukId: UUID): HentBehandlingResponsDto {
```

Comment:
Eg er litt usikker på om eg likar at Dto blir returnert så langt nede i stacken. For meg eksisterer Dto og Mapper nærmare REST-endepunktet.
Bør iallefall ha eit anna navn på metoden, sidan det er ikkje behandling me returnerer, men eit objekt med behandling, fagsak og bruker.

Reply:
Jeg synes det hører hjemme her. For meg så skal ikke rest-endepunkt inneholde noe logikk.

Reply:
Eg er ikkje heilt einig i at mapping domeneobjekt -> Dto er logikk, men...